scripts_python/create_hdf5_dataset.py

```diff
@@ -238,6 +241,8 @@ def _add_all_subjs_to_database(args, chosen_subjs: List[str],
 
                 streamlines_group.attrs['space_attributes'] = \
                     str(tractogram.space_attributes)
+                streamlines_group.attrs['space'] = space
```

Comment:
Isn't this going to save only an `int`? I think we should save a string

Reply:
I don't think it saves an int...

ipython:
```
from dipy.io.stateful_tractogram import Space
a = {}  
a['space'] = Space.RASMM
dict(a)                                                                                                                                                                       
```

Out: {'space': <Space.RASMM: 'rasmm'>}

Reply:
You're right, sorry about that, I don't know why I was thinking about c++ :(